src/MicrosoftTeams.ts

```diff
@@ -151,6 +151,12 @@ namespace microsoftTeams {
     favoriteTeamsOnly?: boolean;
   }
 
+  export const enum TaskModuleDimension {
+    Large = "80px",
+    Medium = "60px",
+    Small = "40px"
```

Comment:
I'm not sure we want to list explicit px sizes here for the enum. I think the Large, Medium, and Small should essentially be snap points that we can choose to map to sizes on our end and change them in the future if needed. Have you verified what this looks like in the message? Do we send over the actual values (i.e. 40, 60, 80) or the enum names (i.e. Small, Medium, Large)?

Reply:
The last time I checked, the spec said these are percentages of the available space. 40, 60 and 80 pixels are *really* small dimensions.